### PR TITLE
Create SEPA Module

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

 - Create SEPA module

 ### Checklist

 - ~[ ] Added a changelog entry~ will add a changelog entry when we open a PR to merge our feature branch into master

- @sarahkoop @jaxdesmarais

### CODE REVIEW

File: SEPADebit/src/androidTest/java/com/braintreepayments/api/SEPADebitClientTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+// TODO: replace with integration tests
```
Comment (cgdibble) [inline SEPADebit/src/androidTest/java/com/braintreepayments/api/SEPADebitClientTest.java:13]: Did you want to resolve this prior to merging?

**Review by cgdibble — APPROVED** [review]
Just want to make sure the TODOs are acceptable, otherwise 👍

File: SEPADebit/src/androidTest/java/com/braintreepayments/api/SEPADebitClientTest.java (hunk 1 above)
Comment (sarahkoop) [reply SEPADebit/src/androidTest/java/com/braintreepayments/api/SEPADebitClientTest.java:13]: This is for future PR work so all good 👍

File: SEPADebit/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (sshropshire) [inline SEPADebit/.gitignore:1]: This file gets auto generated. We can remove since this is covered by the top-level `.gitignore`.

**Review by sshropshire — APPROVED** [review]

**Pull request merged**